pymongo/topology_description.py

```diff
@@ -104,23 +118,23 @@ def _init_incompatible_err(self):
 
             if server_too_new:
                 self._incompatible_err = (
-                    "Server at %s:%d requires wire version %d, but this "
+                    "Server at %s:%d requires wire version %d, but this "  # type: ignore
```

Comment:
Interesting is this type ignore masking an actual bug for unix domiain socket addresses where the port is None?

Reply:
I don't think so, because we have the `s.address[1] or 0` guard, but that isn't being handled by `mypy`.